src/app/actions/BookActions.js

```diff
@@ -2,11 +2,7 @@
 import alt from '../alt';
 
 class Actions {
-  updateListType(listType) {
```

Comment:
@ktp242 the list type is already part of the whole picks data set. Since there are a lot of pieces of information that gets updated, instead of updating each individually, it's better to just update the whole object once and get the data from the set object. So I removed this action and store value.